packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { MultiSigWallet } from "../universal/MultiSigWallet.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @custom:proxied
+ * @title SecurityCouncil
+ * @notice SecurityCouncil receives validation requests for specific output data,
+ *         and allows security council parties to validate & agree on transactions before execution.
+ */
+contract SecurityCouncil is MultiSigWallet, Semver {
+    /**
+     * @notice A mapping of output info array which have been submitted to validate.
+     */
+    mapping (uint => OutputInfo) public outputInfos;
```

Comment:
Could you use `uint256` instead of `uint`?

Reply:
Right, applied.